openlibrary/templates/lib/history.html

```diff
@@ -33,7 +35,7 @@ <h2 class="collapse">
             $ rdf = page.key + ".rdf" + hist_addition
             $ json = page.key + ".json" + hist_addition
             $ opds = page.key + ".opds" + hist_addition
-            <span id="historyTools" class="smaller brown sansserif" style="float:right;padding-top:8px;">
+            <span id="historyTools" class="pageHistory__tools smaller brown sansserif" style="float:right;padding-top:8px;">
```

Comment:
Is there any way we can rm the inline styles here?

Reply:
Done